src/demo-app/sticky-header/sticky-header-demo.html

```diff
@@ -0,0 +1,99 @@
+<div class="demo-sticky">
+  <div class="centered" cdk-scrollable>
+    <div>
+      <h1>Sticky-header Demo</h1>
+    </div>
+
+    <div class="container">
+
+      <div>
+        <div cdkStickyHeader class="headerStyle headerColor1" >
```

Comment:
:+1: Changed to `header-style` and `header-color1` `header-color2` `header-color3`